include/rmm/cuda_stream_view.hpp

```diff
@@ -33,19 +33,31 @@ namespace rmm {
  */
 class cuda_stream_view {
  public:
-  constexpr cuda_stream_view()                                   = default;
-  constexpr cuda_stream_view(cuda_stream_view const&)            = default;
-  constexpr cuda_stream_view(cuda_stream_view&&)                 = default;
+  constexpr cuda_stream_view()                        = default;
+  constexpr cuda_stream_view(cuda_stream_view const&) = default;  ///< Default copy constructor
+  constexpr cuda_stream_view(cuda_stream_view&&)      = default;  ///< Default move constructor
```

Comment:
Let's not write obvious/superfluous documents. The existing generated docs make it pretty clear (to any C++ programmer) that there is a default copy ctor: https://docs.rapids.ai/api/librmm/stable/classrmm_1_1cuda__stream__view.html